flink-core/src/test/java/org/apache/flink/api/common/typeinfo/TypeInformationTest.java

```diff
@@ -61,7 +61,7 @@ public void testOfGenericClassForGenericType() {
 	@Test
 	public void testOfTypeHint() {
 		assertEquals(BasicTypeInfo.STRING_TYPE_INFO, TypeInformation.of(String.class));
-		assertEquals(BasicTypeInfo.STRING_TYPE_INFO, TypeInformation.of(new TypeHint<String>(){}));
+		assertEquals(BasicTypeInfo.STRING_TYPE_INFO, Types.STRING);
```

Comment:
Please undo this one, the test explicitly tests the TypeHint use.

Reply:
OK, sorry maybe batch replace, will revert it